src/database/services/PanelService.ts

```diff
@@ -0,0 +1,47 @@
+import { Panel } from '../models'
+
+interface PanelConfig {
+    image: string,
+    index: number,
+    panel_set_id: number,
+}
+
+
+/**
+ * Perform queries on the 'Panels' table
+ */
+class PanelService {
+
+    /**
+     * Create a new Panel}
+     * @param {} newPanel
+     */
+    static async createPanel(newPanel: PanelConfig){
+        const {id, image, index, panel_set_id } = await Panel?.create({
+            image: newPanel.image,
+            index: newPanel.index,
+            panel_set_id: newPanel.panel_set_id,
+        });
+        return {id, image, index, panel_set_id };
+    }
+
+    /**
+     * Authenticate a panel before retrieve the panel's data
+     * @param id 
+     * @returns 
+     */
+    static async getPanel(id: number) {
```

Comment:
I imagine there will be more ways to get a panel in the future, but this is fine for now

Reply:
yeah, I just did what was on the projects board. We can discuss with Travis what other queries are useful today.